src/mesh/webgl/MeshRenderer.js

```diff
@@ -209,4 +209,7 @@ MeshRenderer.prototype.start = function ()
  */
 MeshRenderer.prototype.destroy = function ()
 {
+    if (this.renderer) {
```

Comment:
Is this check necessary? When is there not a renderer value in this object renderer?